src/main/java/com/minecolonies/coremod/colony/colonyEvents/citizenEvents/CitizenBornEvent.java

```diff
@@ -0,0 +1,59 @@
+package com.minecolonies.coremod.colony.colonyEvents.citizenEvents;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.minecolonies.api.util.constant.Constants;
+
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.util.ResourceLocation;
+
```

Comment:
Something here seems off

Reply:
why? its just three imports and an empty line before the class description.